### DISCUSSION TITLE
StellarQuest Prologue and Chapter 2 by Sweetbread

### DISCUSSION THREAD

**Discussion by sweetbreadx3** [stellar/sorobanathon#173] [General]
hiii
here are my submissions for the prologue and chapter 2 of the quest~
https://litemint.com/items/GBAX725UPDQTLF6NROZC2R5WX32JUZ6GSPS2BK6U6VR62IKK5MBHKYLU/NFT
https://litemint.com/items/GCS4FCE32WZM4CVJSX7K7BNNPPEGRXG3DF27TQSAISEEGRBMSOMFP2G6/NFT

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 600 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
